assets/src/stories-editor/plugins/export.js

```diff
@@ -0,0 +1,73 @@
+/* global fetch */
```

Comment:
We're more and more getting away from using `/* global ... */` annotations in favor of a more explicit way:

```js
const { fetch } = window;
```